### DISCUSSION TITLE
Unable to call the .env file

### DISCUSSION THREAD

**Discussion by QsAdedipe** [Cyfrin/foundry-full-course-cu#1567] [Q&A] [answered]
after setting the .env data in the gitignore file, I am unable to call the .env file. Below is the error I am havving:

root@DESKTOP-IM9H78N:~/foundry-simple-storage-f23# source .env
PRIVATE_KEY: command not found
RPC_URL: command not found

**Comment by EngrPips** [comment]
Hello @QsAdedipe , please run the below command in your terminal after you have run `source .env` and you should get back those values 

```javascript
echo $PRIVATE_KEY
```

```javascript
echo $RPC_URL
```

**Comment by QsAdedipe** [comment]
still saying command not found

On Tue, Apr 2, 2024 at 10:07 PM EngrPips ***@***.***> wrote:

> Hello @QsAdedipe <https://github.com/QsAdedipe> , please run the below
> command in your terminal after you have run source .env and you should
> get back those values
>
> echo $PRIVATE_KEY
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Cyfrin/foundry-full-course-f23/discussions/1567#discussioncomment-8989467>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXKXH2N6T24H7ESHIQJJQIDY3MMYFAVCNFSM6AAAAABFUBYPJCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DSOBZGQ3DO>
> .
> You are receiving this because you were mentioned.Message ID:
> <Cyfrin/foundry-full-course-f23/repo-discussions/1567/comments/8989467@
> github.com>
>


-- 
QS Adedipe Oluwasina, MNIQS, RQS

**Reply by EngrPips** [reply, marked as answer]
Maybe the issue is with your `.env` , did you ensure that there is no space between the `variable` , `=` and the `value`

**Comment by QsAdedipe** [comment]
Thank you, it is working now.


On Wed, Apr 3, 2024 at 9:41 AM EngrPips ***@***.***> wrote:

> Maybe the issue is with your .env , did you ensure that there is no space
> between the variable , = and the value
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Cyfrin/foundry-full-course-f23/discussions/1567#discussioncomment-8994163>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXKXH2NBZP3BKNHZ2NV5Y4DY3O6B5AVCNFSM6AAAAABFUBYPJCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DSOJUGE3DG>
> .
> You are receiving this because you were mentioned.Message ID:
> <Cyfrin/foundry-full-course-f23/repo-discussions/1567/comments/8994163@
> github.com>
>


-- 
QS Adedipe Oluwasina, MNIQS, RQS

**Reply by EngrPips** [reply]
Awesome, I am glad you was able to fix it. keep crushing them!

**Comment by QsAdedipe** [comment]
Thank you

On Wed, Apr 3, 2024 at 11:30 AM EngrPips ***@***.***> wrote:

> Awesome, I am glad you was able to fix it. keep crushing them!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Cyfrin/foundry-full-course-f23/discussions/1567#discussioncomment-8995342>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXKXH2P27ED77UTC4LV5CFTY3PK37AVCNFSM6AAAAABFUBYPJCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DSOJVGM2DE>
> .
> You are receiving this because you modified the open/close state.Message
> ID: <Cyfrin/foundry-full-course-f23/repo-discussions/1567/comments/8995342
> @github.com>
>


-- 
QS Adedipe Oluwasina, MNIQS, RQS

**Reply by EngrPips** [reply]
Anytime buddy
